titan/features/prep.py

```diff
@@ -130,11 +135,11 @@ def get_acquisition_risk_multiplier(self, time: int, interaction_type: str):
         if self.active and self.last_dose_time is not None:
             params = self.agent.location.params
             if self.type == "Oral":
-                if self.adherence == 1:
+                if self.adherent:
                     return 1.0 - params.prep.efficacy.adherent
                 else:
                     return 1.0 - params.prep.efficacy.non_adherent
-            elif self.type == "Inj" and self.adherence == 1:
+            elif self.type == "Inj" and not self.adherent:
```

Comment:
the old code is checking for adherence and the new is checking for non-adherence, is that change purposeful?